content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -44,13 +44,16 @@ You can use `kubectl create configmap` to create a ConfigMap from multiple files
 For example:
 
 ```shell
-kubectl create configmap game-config --from-file=https://k8s.io/docs/tasks/configure-pod-container/configmap/kubectl
+mkdir -p configure-pod-container/configmap/kubectl/
+wget https://k8s.io/configure-pod-container/configmap/kubectl/game.properties -o configure-pod-container/configmap/kubectl/game.properties
```

Comment:
The url here is obviously wrong!

Reply:
Also don't get the idea why we use such a complicated local path for experiments.